tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py27, flake8
+
+[testenv]
+passenv = DISPLAY
+deps =
+    pytest==3.0.3
+    pytest-selenium==1.5.0
+    selenium==3.0.1
+commands = pytest --driver Firefox {posargs}
+
+[testenv:flake8]
+deps =
+	flake8
+commands = flake8 foxpuppet tests
```

Comment:
We should run this on the current directory rather than list this. You can make this overridable using `{posargs:.}`